### DISCUSSION TITLE
"Change email address" email validation and token not sent anymore.

### DISCUSSION THREAD

**Discussion by vianneypercq** [supabase/supabase#28188] [Questions] [answered]
Hello everyone! 
My problem is that when a user changes his email address, he used to receive a validation email on his new address. This corresponded to the option “If disabled, only the new email is required to confirm.” with the “Secure email change” toggle set to Off (cf. screenshot). **This is exactly what I want and it worked absolutely perfectly without any problems.** I could see in the auth users table the token for changing the address until the new address was validated via the link in the email sent. 

**->Today this feature no longer works.** The new email is immediately validated and replaces the old email without the validation email with the token being sent.
(I don't have this problem with password changes, where the validation mail and token are well sent!)

Thanks for helping.

![CleanShot 2024-07-25 at 11 48 49@2x](https://github.com/user-attachments/assets/a99a1d60-7be0-459d-911d-3d7b2cc2bdd7)

**Comment by GaryAustin1** [comment, marked as answer]
I don't know if you are the same user as on Discord... but:
https://discord.com/channels/839993398554656828/1006358244786196510/threads/1265878876538540084
And apparently there is a fix in the work, see towards the end of that thread.

**Comment by vianneypercq** [comment]
Hey @GaryAustin1 Thanks for sharing this Discord thread I was not aware!
My problem is actually in the Github posts mentioned in the Discord thread.

**Comment by vianneypercq** [comment]
A fix has been deployed and everything is back to normal! I've run some tests and changing the email correctly triggers an email to be sent for validation via a token.
